Lombiq.Hosting.Tenants.IdleTenantManagement/Manifest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OrchardCore.Modules.Manifest;
+using static Lombiq.Hosting.Tenants.IdleTenantManagement.Constants.FeatureNames;
+
+[assembly: Module(
+    Name = "Lombiq Hosting - Idle Tenant Management",
+    Author = "Lombiq Technologies",
+    Website = "https://github.com/Lombiq/Hosting-Tenants",
+    Version = "0.0.1",
+    Description = "Managing idle tenants."
+)]
+
+[assembly: Feature(
+    Id = DisableIdleTenants,
+    Name = "Lombiq Hosting - Idle Tenant Management - Disable Idle Tenants",
+    Category = "Hosting",
+    Priority = "9999",
```

Comment:
Is this needed?

Reply:
Yes this one is enabling the feature.

Reply:
The priority?

Reply:
The backgroundTask should be the last to run amongst the bg task because if it kills the shell prior to everything else finish it could cause problems in O1 there was also some settings that set the priority to last.